README.md

```diff
@@ -4,6 +4,12 @@ The Student Robotics public documentation.
 
 ## Getting Started
 
+For ease of setup, a Docker container is provided. Simply install Docker and `docker-compose`, and run `docker-compose up`.
```

Comment:
Suggest that we not put the docker setup first given that installing and setting up docker is a) not detail here at all (unlike the manual docs which include links and guidance for setup), and b) not at all easy (given that it usually involves messing with user group membership and accepting the security trade-offs involved in running docker locally).

Reply:
I think it's a good thing not documenting how to instal docker, as there's nothing project specific in it, and it varies by platform anyway.

As for the ease, it's really not bad nowadays. It's in most package managers, and the desktop version is meant to be simple to use. Things do often get run as `root` however, yes.

Reply:
> I think it's a good thing not documenting how to instal docker, as there's nothing project specific in it, and it varies by platform anyway.
> 
> As for the ease, it's really not bad nowadays. It's in most package managers, and the desktop version is meant to be simple to use. Things do often get run as `root` however, yes.

This may be the case, however there's no links or anything. Therefore for people who are most likely to benefit from not installing Ruby (i.e: people who aren't developers) they're dropped in the deep end.